tests/multipart.cpp

```diff
@@ -178,4 +178,39 @@ TEST_CASE("multipart legacy test", "[multipart]")
     assert(received.empty());
     assert(str == "One-hundred");
 }
+
+TEST_CASE("multipart codec", "[multipart]")
```

Comment:
This seems to contain two test cases that somehow depend on each other. Please split this up, and consider this when adding further test cases.

Reply:
Will do.  They are actually independent.  I'll add the others tests suggested as individual tests.